samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+********
+
+This application demonstrates BLE peripheral by exposing vendor-specific
+GATT services. Currently only button notification and LED service are
+implemented. Other BLE sensor services can easily be added.
+See `BlueST protocol`_ document for details of how to add a new service.
+
+Instructions for `ST Nucleo WB55RG`_ board
```

Comment:
Please delete this section. This is not specific to this sample and valid for all BLE samples for this board.
If missing details on this board, this should be added directly in board doc.

Reply:
ok